### ISSUE TITLE
--lowercase ?

### ISSUE THREAD

**Issue by schoening** [mstssk/sw2dts#38]
Hi. An issue I frequently run into is that Swagger is using UpperCased property names, but the server transforms the actual json into lowerCase. I can transform the Json before using it, but I would much rather save a step and have a lowercase / uppercase parameter in this great tool :)

**Comment by mstssk** [comment]
Thank you for your issue @schoening .

I understood in this way, is it correct? ↓

```json
{
    "definitions": {
        "Dummy": {
            "type": "object",
            "properties": {
                "PascalCaseProp": {
                    "type": "string"
                }
            }
        }
    }
}
```

Above json is converted into following d.ts normally.

```ts
export interface Dummy {
    PascalCaseProp?: string;
}
```

However you want different d.ts like below:

```ts
export interface Dummy {
    pascalCaseProp?: string;
}
```

**Comment by schoening** [comment]
@mstssk exactly. I cannot speak for anyone else but the last 2 jobs using .NET had this quirk where swagger would use their PascalCase but the actual API would return "camelCased" json.

**Comment by schoening** [comment]
Actually, "Dummy" would also be lowercased. Not the interfacename, but in the JSON.
